### PR TITLE
Changing URL from graphql-manual to graphql only

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-graphql-4.surge.sh

File: preview.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ content:
 
 ui:
   bundle:
-    url: https://s3-eu-west-1.amazonaws.com/static-content.neo4j.com/build/ui-bundle-latest.zip
+    url: https://d12wh7zj8x3amw.cloudfront.net/build/ui-bundle-latest.zip
```
Comment (nvitucci) [inline preview.yml:17]: Do we consider this relatively stable now?

Comment (lidiazuin) [reply preview.yml:17]: Idk, it was the link that Adam sent yesterday

Comment (recrwplay) [reply preview.yml:17]: If you don't mind the possibility of updating again later, it's fine to use this. I can't say 100% that we won't offer a different bundle URL soon, but at least this one works

Comment (lidiazuin) [reply preview.yml:17]: Yep it's ok, I'm still working on other things in this repo anyway!

**Review by nvitucci — APPROVED** [review]
I am ok with this change, as long as redirects are also put in place. Anyway, this should not be a one-off since we follow the same URI structure in other repos too.

@davidoliverSP2 @recrwplay What do you think?

File: preview.yml (hunk 1 above)
Comment (nvitucci) [reply preview.yml:17]: Sounds good.

**Comment by lidiazuin** [comment]
> I am ok with this change, as long as redirects are also put in place. Anyway, this should not be a one-off since we follow the same URI structure in other repos too.
> 
> @davidoliverSP2 @recrwplay What do you think?

There are a few repos that are still using manual in the URL like cypher and drivers

**Pull request merged**